gql/createPost.gql

```diff
@@ -0,0 +1,9 @@
+mutation createPost($title: String!, $url: String!) {
```

Comment:
You are right, component based is better. And im looking for better folders structure too (I take inspiration from https://github.com/scalable-react/scalable-react-typescript-boilerplate and https://github.com/devinit/datahub). I hope I will show it later